src/js/scripts.js

```diff
@@ -79,7 +79,10 @@ import msToMinutesAndSeconds from '../modules/modal-timeout/utils/msToMinutesAnd
             }
         }
 
-        if (modalElements.length) {
+        sessionData = getSessionData();
+        const sessionEnded = !sessionData.alive;
```

Comment:
remove this

Reply:
Refactored the code and made new improvements which we discussed yesterday.